packages/angular_devkit/build_angular/src/utils/routes-extractor/BUILD.bazel

```diff
@@ -0,0 +1,27 @@
+# Copyright Google Inc. All Rights Reserved.
+#
+# Use of this source code is governed by an MIT-style license that can be
+# found in the LICENSE file at https://angular.io/license
+
+load("//tools:defaults.bzl", "ts_library")
+
+# NOTE This is built as ESM as this is included in the users server bundle.
```

Comment:
Is this using a different tsconfig to all ESM output?  The main tsconfig is set to `commonjs` currently.

Reply:
Under bazel, the "module" and "target" are not configured in the tsconfig.

[is](https://github.com/angular/angular-cli/blob/main/tools/defaults.bzl#L54-L89)